src/core/public/core_app/status/components/metric_tiles.tsx

```diff
@@ -7,24 +7,99 @@
  */
 
 import React, { FunctionComponent } from 'react';
-import { EuiFlexGrid, EuiFlexItem, EuiCard } from '@elastic/eui';
-import { formatNumber, Metric } from '../lib';
+import { EuiFlexGrid, EuiFlexItem, EuiCard, EuiStat } from '@elastic/eui';
+import { DataType, formatNumber, Metric } from '../lib';
 
 /*
- * Displays a metric with the correct format.
+ * Displays metadata for a metric.
  */
-export const MetricTile: FunctionComponent<{ metric: Metric }> = ({ metric }) => {
-  const { name } = metric;
+export const MetricCardFooter: FunctionComponent<{
+  testSubjectName: string;
```

Comment:
nit: you can probably call this `data-test-subj` directly (as EUI does)

Reply:
I thought it wasn't possible to spread such keys, but in case someone else though so, TIL:

```ts
const foo = { 'data-test-subj': 'bar' };
const { 'data-test-subj': dataTestSub } = foo;
```